builtin/game/falling.lua

```diff
@@ -6,6 +6,34 @@ local builtin_shared = ...
 -- Falling stuff
 --
 
+local node_fall_hurt = minetest.setting_getbool("node_fall_hurt") ~= false
+
+local function fall_hurt_check(self, pos, dtime)
+	if not node_fall_hurt then return end
+	if self.hurt_timer > 0 then
+		self.hurt_timer = self.hurt_timer - dtime
+		return
+	end
+
+	-- Get damage level from falling_node_damage group
+	local damage = core.registered_nodes[self.node.name] and
```

Comment:
as this won't change, you could cache this in the entity to avoid a global look up and also the calling of fall_hurt_check at all